s4e-web/src/app/views/settings/profile/profile.component.html

```diff
@@ -1,33 +1,85 @@
+      <img src="./assets/images/ico_people.svg" alt="Ikona ludzi" width="50" tile-image />
+      <header class="panel__header" tile-title>Moje dane</header>
+      <div tile-description>
+        <p i18n>Imie: Jan</p>
+        <p i18n>Nazwisko: Kowalski</p>
+        <p i18n>Email: {{ userEmail$ | async }}</p>
+      </div>
+      <button
+        class="button button--secondary button--small"
+        routerLink="/settings/change-password"
+        queryParamsHandling="merge"
+        data-e2e="go-to-password-change"
+        footer-navigation
+      >
+        Zmień hasło
+      </button>
+    </s4e-tile>
 
-<section class="content__details content__details--50 profile">
-  <div
-    class="profile__content"
-    data-e2e="user-details"
-  >
-    <p *ngIf="!(isLoggedIn$ | async); else userDetails" i18n>
-      Zaloguj się by zobaczyć swój profil
-    </p>
-    <ng-template #userDetails>
-      <h2>Jan Kowalski</h2>
-      <p i18n>Email: {{ userEmail$ | async }}</p>
-      <p>
-        Jesteś administratorem instytucji <a href="#">Ochotnicza Straż Pożarna, Kraków i 5 innych</a>
-      </p>
-    </ng-template>
+    <s4e-tile
+      *ngIf="(administrationInstitutions$ | async).length > 0"
+      data-ut="administrative-institutions-tile"
+    >
+      <img src="./assets/images/ico_institution.svg" alt="Ikona instytucji" width="50" tile-image />
+      <header class="panel__header" tile-title>Instytucje administrowane</header>
+      <div tile-description>
+        <p *ngFor="let institution of (administrationInstitutions$ | async).slice(0, MAX_INSTITUTIONS_TO_DISPLAY_PER_TILE)">
+          <a
+            routerLink="/settings/institution"
+            [queryParams]="{institution: institution.slug}"
+            queryParamsHandling="merge"
+          >
+            {{ institution.name }}
+          </a>
+        </p>
+      </div>
+      <button
+        *ngIf="hasMoreAdministrationInstitutionsThanMax$ | async"
+        class="button button--secondary button--small"
+        routerLink="/settings/institutions"
+        queryParamsHandling="merge"
+        data-ut="go-to-institutions"
+        data-e2e="go-to-institutions"
+        footer-navigation
+      >
+        Pokaż więcej
+      </button>
+    </s4e-tile>
 
-  </div>
-  <footer>
-    <button
-      class="button button--secondary button--small"
-      routerLink="/settings/change-password"
-      queryParamsHandling="merge"
-      data-e2e="go-to-password-change"
+    <s4e-tile
+      *ngIf="(memberInstitutions$ | async).length > 0"
+      data-ut="member-institutions-tile"
     >
-      Zmień hasło
-    </button>
-  </footer>
-</section>
+      <img src="./assets/images/ico_institution.svg" alt="Ikona instytucji" width="50" tile-image />
+      <header class="panel__header" tile-title>Instytucje członkowskie</header>
+      <div tile-description>
+        <p *ngFor="let institution of (memberInstitutions$ | async).slice(0, MAX_INSTITUTIONS_TO_DISPLAY_PER_TILE)">
```

Comment:
This is something that should not be done (calling functions on collections which are iterated in template)
this slice function will be called on every refresh cycle and whole list will be rerendered.
I would suggest adding this slice operation to the pipe of `memberInstitutions$` observable.